lib/meterReadingUtils.js

```diff
@@ -0,0 +1,17 @@
+import { getMeterReadingsandInvoicesByIds } from '../airtable/request';
+
+export const getLatestMeterReadingForCustomer = async (customer) => {
```

Comment:
Maybe add a comment to mention the different cases where this would return `null`